src/unlimited.cpp

```diff
@@ -1423,6 +1697,10 @@ static const CRPCCommand commands[] =
     { "util",               "getstat",                &getstat,                true  },
     { "util",               "get",                    &gettweak,               true  },
     { "util",               "set",                    &settweak,               true  },
+
+    /* Coin generation */
+    { "generating",         "getgenerate",            &getgenerate,            true  },
+    { "generating",         "setgenerate",            &setgenerate,            true  },
```

Comment:
It was a leftover, it is suppose to be only in `unlimited.cpp`. 
I'm going to remove it from `rpc/mining.cpp`,  retest and then update the PR.